k8s/us-central1/gcs-buckets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
so this gets pushed when we run `kubectl apply -f k8s/` and then other files can refer to it using:

```
              envFrom: [
                {
                  configMapRef: {
                    name: "gcs-buckets",
                  },
                },
              ],
```

Is that right? The reference is just based on `metadata.name` ? Or is there some other step to allow the other files to refer to this config yaml?

Reply:
The config map is referenced by name. I deployed it manually this time to the right namespace (`--namespace=automated`), and I'll follow up with another change to update the postsubmit script.